src/website/shared/templates/base.html

```diff
@@ -29,6 +29,7 @@ <h1>
         {% block auth %}
         {% if user.is_authenticated %}
         <li>
+          <span>{{user.username}}</span>
```

Comment:
```suggestion
          {% if user.provider == "github" %}
            <a href="https://github.com/{{ user.username }}" target="_blank">{{ user.username }}</a>
          {% else %}
            <span>{{ user.username }}</span>
          {% endif %}
```

What we we do something like this, but corrected for the right login name from socialaccounts?

Reply:
Amended this approach in the commit